repl/src/main/scala/com/cloudera/livy/repl/Session.scala

```diff
@@ -238,25 +237,4 @@ class Session(
 
     compact(render(resultInJson))
   }
-
-  private def setJobGroup(statementId: Int): String = {
```

Comment:
Instead of using String statement to set job group we use underlying SparkContext in each interpreter to directly set job group.